server/cli/src/cli.rs

```diff
@@ -396,6 +398,77 @@ async fn main() -> anyhow::Result<()> {
             info!("Refresh data result: {:#?}", result);
         }
         Action::SignPlugin { path, key, cert } => sign_plugin(&path, &key, &cert)?,
+        Action::UpsertStandardReports => {
+            let connection_manager = get_storage_connection_manager(&settings.database);
+            let con = connection_manager.connection()?;
+            let base_reports_dir = "./reports";
+
+            let report_names = fs::read_dir(base_reports_dir)?
+                .map(|res| res.map(|e| e.path().into_os_string().into_string().unwrap()))
+                .collect::<Result<Vec<String>, std::io::Error>>()?;
+
+            for name_dir in report_names {
+                let report_versions = fs::read_dir(&name_dir)?
+                    .map(|res| res.map(|e| e.path().into_os_string().into_string().unwrap()))
+                    .collect::<Result<Vec<String>, std::io::Error>>()?;
+
+                let (_, name) = name_dir.rsplit_once('/').unwrap();
+
+                for version_dir in report_versions {
+                    let (_, version) = version_dir.rsplit_once('/').unwrap();
+
+                    let arguments_path = format!("{version_dir}/arguments.json");
+                    let arguments_ui_path = format!("{version_dir}/arguments_ui.json");
+
+                    let manifest_file = fs::File::open(format!("{version_dir}/manifest.json"))
+                        .expect("file should open read only");
```

Comment:
```suggestion
                        .expect("Can't open manifest for report {} {}", reportname, version);
```
Might be helpfull to make this error a bit more explicit?